.github/workflows/deploy.yaml

```diff
@@ -420,15 +421,16 @@ jobs:
 
       - name: 'Register participant'
         run: |
-          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.registrationservice:registration-service-cli:0.0.1-SNAPSHOT:jar:all -DoutputDirectory=.
-          java -jar registration-service-cli-0.0.1-SNAPSHOT-all.jar \
-            -s $REGISTRATION_SERVICE_API_URL \
-            -d did:web:$DID_HOST \
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.registrationservice:registration-service-cli:$REGISTRATION_SERVICE_VERSION:jar:all -DoutputDirectory=.
+          java -jar registration-service-cli-$REGISTRATION_SERVICE_VERSION-all.jar \
+            -d did:web:$DATASPACE_DID_HOST \
+            -c did:web:$DID_HOST \
```

Comment:
What abou renaming DID_HOST to something more descriptive, eg PARTICIPANT_ DID_HOST as we have now DID_HOST and DATASPACE_DID_HOST?